python/lsst/sims/catUtils/utils/SNIaLightCurveGenerator.py

```diff
@@ -56,6 +57,20 @@ def __init__(self, *args, **kwargs):
         self._brightness_name = 'flux'
         super(SNIaLightCurveGenerator, self).__init__(*args, **kwargs)
 
+    def light_curves_from_pointings(self, pointings, chunk_size=100000, lc_per_field=None):
+        if lc_per_field is not None:
+            warnings.warn("You have set lc_per_field in the SNIaLightCurveGenerator. "
+                          "This will limit the number of candidate galaxies queried from the "
+                          "CatSim database per field-of-view.  Because supernovae are randomly "
+                          "populated in those galaxies, there is no guarantee that the galaxies "
+                          "queried will have supernovae in them.  If the galaxies you actuall "
+                          "query do not host supernovae, you could get fewer light curves than "
+                          "you expect.")
+
```

Comment:
Do we have a way to actually ask and get a certain number of lightcurves with SNe in a pointing? Or plans to?